core/services/standardcapability/standard_capability.go

```diff
@@ -0,0 +1,93 @@
+package standardcapability
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/loop"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/job"
+	"github.com/smartcontractkit/chainlink/v2/plugins"
+)
+
+type StandardCapability struct {
+	log                logger.Logger
+	spec               *job.StandardCapabilitySpec
+	pluginRegistrar    plugins.RegistrarConfig
+	telemetryService   core.TelemetryService
+	store              core.KeyValueStore
+	capabilityRegistry core.CapabilitiesRegistry
+	errorLog           core.ErrorLog
+	pipelineRunner     core.PipelineRunnerService
+	relayerSet         core.RelayerSet
+
+	capabilityLoop *loop.StandardCapabilityService
+}
+
+func NewStandardCapability(log logger.Logger, spec *job.StandardCapabilitySpec,
```

Comment:
@ettec I think most of these fields can be private: NewStandardCapability -> newStandardCapability; StandardCapability -> standardCapability etc